devices.js

```diff
@@ -5139,6 +5139,14 @@ const devices = [
         extend: ledvance.light_onoff_brightness_colortemp_colorxy,
         ota: ota.ledvance,
     },
+    {
+        zigbeeModel: ['A60 RGBW Value II'],
+        model: 'AC25697',
+        vendor: 'LEDVANCE',
+        description: 'SMART+ classic E27 RGBW',
+        extend: ledvance.light_onoff_brightness_colortemp_colorhs,
```

Comment:
Why did you use `colorhs` here and not `colorxy` (which is preferred)?

Reply:
Simply due to ignorance... I'll change it and notify you (resolve conversation) if done.
Unfortunatelly I can (re-) test the correct functionality soonest mid of january... (If you're sure that it will no interfere with the current code, we could change it on the fly without testing).

Sorry for the delayed reply, stress at work and christmas preparations (merry xmas btw!)....